docs/docsite/rst/getting_started_ee/introduction.rst

```diff
@@ -0,0 +1,65 @@
+.. _introduction_execution_environments:
+
+######################################
+Introduction to execution environments
+######################################
+
+Ansible execution environments aim to resolve complexity issues and provide all the benefits you can get from containerization.
+
+Reducing complexity
+===================
+
+There are three main areas where EEs can reduce complexity: software dependencies, portability, and content separation.
+
+Dependencies
+------------
+
+Software applications typically have dependencies.
+It can be software libraries, configuration files or other services, to name a few, and Ansible, being an exceptional automation tool, is not an exception in terms of the mentioned.
+
+Traditionally, application dependencies are installed by administrators on top of an operating system using packaging management tools like RPM or Python-pip.
+
+The major drawback of such approach is that an application might require versions of dependencies different from those provided by default.
+
+In case of Ansible, a typical installation consists of ``ansible-core`` and a set of Ansible collections.
+
+At present, there are more than a hundred collections included in the Ansible package and hundreds more are available on Ansible Galaxy and Automation Hub for manual installation.
```

Comment:
```suggestion
At present, the Ansible package includes more than a hundred collections included  and you can manually install hundreds more are from  Ansible Galaxy..
```

Reply:
I'm a little worried about talking about the 100s of collections in the package here. I though someone tried to create an EE with the package (aka all those collections) a year or so back and it failed because of too many conflicting dependencies?

Reply:
Fair point, i removed this paragraph not to give people such a "bright" idea:)